source/memory.tex

```diff
@@ -5082,6 +5058,36 @@
 
 \rSec2[smartptr.adapt]{Smart pointer adaptors}
 
+\rSec3[smartptr.adapt.defns]{Definitions}
+
+\pnum
+\indextext{pointer_of@\tcode{\placeholder{POINTER_OF}}}%
+\indexlibrary{pointer_of@\tcode{\placeholder{POINTER_OF}}}%
```

Comment:
The text for the index of library names says "Constructions whose name appears in italics are for exposition only."

so we seem to have resolved to put exposition-only stuff there (at least more widely used things).

Reply:
There should be a record of that decision somewhere...